Vwrs-revamp-first-stage (#1203)
Macrometacorp/docs: docs/photoniq/vwrs/02-get-started-vwrs/index.md

Let's create our first virtual waiting room service with the VWRs GUI to get started.

## Objectives

This quickstart will guide you through:

- Creating and configuring your first VWR with the PhotonIQ VWRS GUI
- Configuring additional waiting room details
- Monitoring and observing the performance of the VWR

## Creating your Virtual Waiting Room

### Steps

To create a waiting room:

1. Login to the VWRS GUI with your Macrometa credentials. Macrometa provides these credentials after you sign up for the service. 
1. Click **Waiting Rooms** from the top nav bar.
1. Click **New Waiting Room**. This opens a pop-up section to enter the following details to set up your virtual waiting room.

> - **waitingroom_name:** The name of the waiting room. Names should be without spaces. Replace spaces with hyphens or dashes.

Reviewer: ```suggestion
- **waitingroom_name:** The name of the waiting room. Names must not have spaces. 
```

    - Must be a number greater than zero.
    
    Note: **For access_type of users and rps, this is the maximum request rate. For users_per_period, the maximum rate is calculated using the active rate divided by the period**

- **max_origin_usage_time**: The time (in seconds) that users can access the origin after being granted access. After this period, users are forwarded to the waiting room again.
- **waiting_room_interval**: Defined in seconds, this interval determines how often the waiting room HTML page is updated with information like the request's position, estimated waiting time, and the maximum number of requests in the waiting room.

![create virtual waiting room](/img/photoniq/vwr/create-vwr.png)

4. Click **Add Waiting Room**

### Configuring Additional Waiting Room Fields

The Virtual waiting room service allows you to further define and guide the behavior of your waiting room with its **Advanced Settings** fields. These fields include the queue type, dequeue mode, and others.

1. Click **Advanced Settings** from the waiting room definition page

2. Configure these additional details:

![additional details](/img/photoniq/vwr/advanced-settings.png)

> - **queue_type**: Defines how requests should be removed from the waiting room. The three possible queue types are fifo, random, and lottery. If this is not set, then the default queue is fifo.

Reviewer: Isn't fifo usually FIFO since it stands for "first in first out"?

- **queue_mode**: You can configure the waiting room to be enabled dynamically. When set to auto, the waiting room is enabled after reaching the defined rate_limit for a specific `metric_interval`. If set to manual, then the waiting room is always enabled.
- **dequeue_mode**: When the value is on, users in the waiting room are granted access to the origin at a rate given by `rate_limit`. When the value is off, users remain in the waiting room indefinitely.

> - **metric_interval**: The time (in seconds) to enable and disable the waiting room. It represents how long the traffic must be at or above the rate limit before being directed to the waiting room.

Reviewer 2: @joubert-macrometa should we update this?

Author: I updated and avoided listing random and priority as queues. What other updates are needed? I know there's going to be a change in UI, but it has yet to be released. If that's the update, I'll update once the new UI is out.
